src/pages/Home/Home.tsx

```diff
@@ -19,7 +19,10 @@ const Home = () => {
   return (
     <>
       <section className={styles.heroSection}>
-        <img className={styles.heroImage} src={heroImage} alt="hero image" />
+        <picture>
+          <source type="image/webp" className={styles.heroImage} />
+          <img className={styles.heroImage} src={heroImage} alt="hero" />
+        </picture>
```

Comment:
```suggestion
          <img className={styles.heroImage} src={heroImage} alt="hero" />
```

webp 이미지만 사용한다면 picture 태그를 사용하지 않아도 될 거 같네요. picture 태그는 webp를 지원하지 않는 환경에 다른 이미지 포맷을 보여주기 위해 사용할 수 있겠네요!

Reply:
헉.. 사실 다른 파일도 고려하다가 결국 한가지만 사용하게 된 걸 반영을 안했네요..🥲
수정하겠습니다~!